docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -406,29 +406,18 @@ this person as a dependent.
 
 **For simulant under 18:**
 
-- Child is a biological or adopted child to reference person 
+- Child is a biological, adopted, foster or step child to reference person 
     * Assign reference person 
-- Child is foster child 
-    * Assign reference person 
-- Child is step child 
-    * If there is a spouse / partner of reference person assign them 
-    * Otherwise assign the reference person 
-- Child is grandchild of reference person 
-    * If there is a child (biological, adopted, foster) of reference person, assign them (if multiple, assign at random) 
-    * Otherwise assign reference person 
-- Child is brother/sister to reference person 
-    * If there is a parent of the reference person, assign them 
-    * Otherwise assign the reference person 
-- Child is other relative to reference person 
-    * Assign a relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, select one of the same race/ethnicity. If there are multiple of the same race/ethnicity, assign at random. 
+- Child is any other relative to reference person (NOT roommate/housemate or other nonrelative)
+    * Assign a relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, assign at random. 
     * If there is not a relative of the appropriate age available, assign the reference person 
 - Child is non-relative (roommate or other nonrelative) to reference person 
-    * Assign another non-relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, select one of the same race/ethnicity. If there are multiple with the same race/ethnicity, assign at random.  
+    * Assign another non-relative of the reference person who is between 20 and 45 years older than the child. If there are multiple, assign at random. 
```

Comment:
similar to above, i think you mean someone who lives in the same household as the child, who does not fall into the category of "relative" --- is that correct?

Reply:
Added more definition here as well - let me know if it makes sense.